### PR TITLE
fixed bug with SRM servtype handling

### PR DESCRIPTION
**Pull request by vrdel**

### CODE REVIEW

**Comment by skanct** [comment]
@kaggis, @pkoro can you please have a look?

File: bin/topology-vo-connector.py [hunk 1]
```diff
@@ -32,7 +32,9 @@
 import re
 import sys
 import xml.dom.minidom
+import copy
 
+LegMapServType = {'SRM' : 'SRMv2', 'SRMv2': 'SRM'}
```
Comment (pkoro) [inline bin/topology-vo-connector.py]: @vrdel maybe I am saying something silly. Do we need this reverse mapping also in downtimes-gocdb-connector and topology-gocdb-connector scripts?

Comment (vrdel) [inline bin/topology-vo-connector.py]: no because SRMv2 service type is not registered in GOCDB

Comment (pkoro) [inline bin/topology-vo-connector.py]: ok got it. Thanks!

**Pull request merged**